docs/standard/architecture/introduction/toc.md

```diff
@@ -0,0 +1 @@
+# [Introduction](index.md)
```

Comment:
I'd remove this TOC file and just have the topic linked from the main sub-TOC directly. And I'd do the same for other TOC files.